config_child_item.go

```diff
@@ -0,0 +1,7 @@
+package libyaml
+
+// ConfigChildItem represents a subitem.  This is normally found in select_one and select_many types.
+type ConfigChildItem struct {
+	Name  string `yaml:"name" json:"name" validate:"required"`
+	Title string `yaml:"title" json:"title"`
```

Comment:
The attributes that have values set to `this.props` are all getting values from the parent item.  Only the title and name are used in a child item.